app/src/main/java/com/example/dynamicdiet/dto/User.kt

```diff
@@ -1,12 +1,12 @@
 package com.example.dynamicdiet.dto
 
-import java.util.*
 import androidx.room.Entity
 import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
 @Entity(tableName="user")
-data class User(var userName: String, var weight: List<Weight>, var lastName: String, var firstName: String, var age: String, @PrimaryKey @SerializedName("id") var userId:Int = 0) {
+//changed age to integer
```

Comment:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.